lib/presentation/speed_dial/my_speed_dial.dart

```diff
@@ -302,9 +325,22 @@ class MySpeedDial extends StatelessWidget {
       animType: AnimType.SCALE,
       title: title,
       desc: desc,
-      autoHide: Duration(milliseconds: 3500),
+//      autoHide: Duration(milliseconds: 3500),
       btnOkOnPress: () {},
-      btnOkColor: color,
+      btnOkColor: Colors.grey,
```

Comment:
```suggestion
      btnOkColor: Colors.red,
```
Closeボタンの色の変更をよろしくお願いします。
色はredではなくても良いです！

Reply:
2e5c2ee
にて修正しました！ご確認お願いします！